### PR TITLE
Fix balancer strategy do not save & load

### PR DESCRIPTION
**Pull request by zzndb**
~~The complex editor still do not load strategy after launch.~~
~~Because of unfamiliar with Qt, I couldn't fix it by the way.(ಥ_ಥ)~~
<(=ｏ^_^)o

### CODE REVIEW

**Comment by DuckSoft** [comment]
Scheduled.

**Comment by dyhkwong** [comment]
https://github.com/Qv2ray/Qv2ray/blob/a3b437a5821fc54f4455a457e1c67be2b83c7949/src/ui/widgets/node/widgets/BalancerWidget.cpp#L96
try replacing `currentIndexChanged(const QString &arg1)` with `currentIndexChanged(int arg1)` here

File: src/ui/widgets/node/NodeDispatcher.cpp [hunk 1]
```diff
@@ -65,7 +65,9 @@ void NodeDispatcher::LoadFullConfig(const CONFIGROOT &root)
         {
             selector << item.toString();
         }
-        const auto meta = make_balancer_outbound(selector, balancer.toObject()["tag"].toString());
+        QString strategyType = balancer.toObject()["strategy"].toObject()["type"].toString();
+        strategyType = strategyType == "" ? "random" : strategyType;
```
Comment (DuckSoft) [inline src/ui/widgets/node/NodeDispatcher.cpp:69]: 这里应该可以用 toString(defaultValue) 那个方法简化一下

Comment (DuckSoft) [inline src/ui/widgets/node/NodeDispatcher.cpp:69]:
```suggestion
        QString strategyType = balancer.toObject()["strategy"].toObject()["type"].toString("random");
```

Comment (zzndb) [reply src/ui/widgets/node/NodeDispatcher.cpp]: 🤣（好像暴露了什么

Comment (DuckSoft) [reply src/ui/widgets/node/NodeDispatcher.cpp]: 诶，写串行了

**Comment by DuckSoft** [comment]
Is this Pull Request ready for review?

**Review by DuckSoft — APPROVED** [review]
Tested. Fixed.
Thanks!

**Pull request merged**